### ISSUE TITLE
How to convert ColumnPtr object to ColumnUInt64 * object?

### ISSUE THREAD

**Issue by 744570676** [ClickHouse/ClickHouse#15431]
I define a class A, which has a member `ColumnUInt64 * col` in A.h file. Then, in A.cpp, I initialize the member `col` in the way:
```
col = ColumnUInt64::create();
```
and I got a error
```
can not convert COWHelper<DB::ColumnVectorHelper, DB::ColumnVector<long unsigned int> >::MutablePtr {aka ‘COW<DB::IColumn>::mutable_ptr<DB::ColumnVector<long unsigned int> >’} to non-scalar std::__1::unique_ptr<DB::ColumnVector<long unsigned int> >
```

**Comment by Sasasu** [comment]
`ColumnUInt64 *col = ColumnUInt64::create().detach();` if you realy want.

**Comment by 744570676** [comment]
> `ColumnUInt64 *col = ColumnUInt64::create().detach();` if you realy want.

ok, thank you~ :)

**Comment by alexey-milovidov** [comment]
@Sasasu @744570676 I'm afraid that's not the best answer.
Because it will release the ownership of the object.

Just write `col.get()` to obtain non-owning pointer.
